lib/components/Editor/CustomExtensions/Markdown/index.js

```diff
@@ -22,9 +28,10 @@ const MarkdownEditor = ({ editor, className, ...otherProps }) => {
   useLayoutEffect(() => {
     const BORDER_HEIGHT = 1;
```

Comment:
Since this doesn't change during run time, let's declare this constant outside the body of the hook.